data/templates/ignition.json

```diff
@@ -0,0 +1,11 @@
+{
+  "ignition": { "version": "2.0.0" },
+  "storage": {
+    "files": [{
+      "filesystem": "root",
+      "path": "/root/ignition-RackHD.txt",
+      "mode":400,
+      "contents": { "source": "data:text/plain;charset=utf-8;base64,WW91IG1heSBjdXN0b21pemUgeW91ciBDb3JlT1MgaW5zdGFsbGF0aW9uIHVzaW5nIElnbml0aW9uIGJ5IGNvbmZpZ3VyaW5nIHRoZSBpZ25pdGlvblNjcmlwdFVyaSB2YXJpYWJsZSB0byBwb2ludCB0byBhIGN1c3RvbSBJZ25pdGlvbiBmaWxlIHByaW9yIHRvIGV4ZWN1dGluZyB0aGUgUmFja0hEIFRhc2suT3MuSW5zdGFsbC5Db3JlT1MgdGFzay4=" }
```

Comment:
@tgelter : 

Thanks for you explaintation.
my suggestion is adding some comment to describe what this base64 encoded string means. It's a pity that JSON doesn't support comment, but I see the ejs supports, please check this help document: https://www.npmjs.com/package/ejs

The example is:
```javascript
{
    ...
   "mode": 400,
   "contents": {
     <%#  The source is encoded with base64, after decoded, it says: %>
     <%#  You may customize your CoreOS installation using Ignition by configuring the ignitionScriptUri variable to point to a custom Ignition file prior to executing the RackHD Task.Os.Install.CoreOS task. %>
     <%#  If you want to change the source, you can use this command tool: xxxx %>
      "source": "data:test/plain;charset=utf-8;base64,xxxxxxxxxxxxxxxxxxxxxxxx"
   }
}
```